web/src/scenes/team/MemberList/RemoveMemberDialog/index.tsx

```diff
@@ -41,7 +41,7 @@ export const RemoveMemberDialog = memo(function RemoveMemberDialog(
 
           <span className="text-14">
             Are you sure you want to remove your team member
-            <b>&nbsp;{memberForRemove?.name}</b>? This will delete their
+            <b>&nbsp;{memberForRemove?.user.name}</b>? This will delete their
```

Comment:
remove "This will delete their Developer Portal account." this is no longer the case